website/mails/presends.py

```diff
@@ -32,6 +32,23 @@ def new_public_project(email):
     public = email.find_sent_of_same_type_and_user()
     return node.is_public and not len(public)
 
+def prereg_reminder(email):
+    """ Check make sure the draft still exists, has not already received a reminder,
+     and has not been submitted.
+
+    :param email: QueuedMail object, with the 'draft_id' in its data field
+    :return: boolean based on whether the email should be sent
+    """
+    # In line import to prevent circular importing
+    from osf.models.queued_mail import QueuedMail
+    from osf.models import DraftRegistration
+
+    draft_id = email.data['draft_id']
+    draft = DraftRegistration.load(draft_id)
+    reminders_sent = QueuedMail.objects.filter(data__draft_id=draft_id).exclude(sent_at=None)
```

Comment:
.exists() is generally more efficient when you don't need to do anything with the queryset
`QueuedMail.objects.filter(data__draft_id=draft_id).exclude(sent_at=None).exists()`

From django queryset docs on exists:
[In many cases it will perform better than a subquery since the database is able to stop evaluation of the subquery when a first matching row is found](https://docs.djangoproject.com/en/1.11/ref/models/expressions/#exists-subqueries)

and then...